tests/test_yaml.py

```diff
@@ -204,3 +204,22 @@ def test_toc(self):
                             'example.example.Foo'
                         )
                         break
+
+    def test_examples(self):
+        """
+        Test second level toc nesting
+        """
+        with sphinx_build('pyexample'):
+            with open('_build/text/docfx_yaml/example.nap.Base.yml') as yml_file:
+                data = yaml.safe_load(yml_file)
+                for item in data['items']:
+                    if item['uid'] == 'example.nap.Base.ref':
+                        self.assertEqual(
+                            item['syntax']['example'].split('\n')[2],
```

Comment:
>item['syntax']['example'].split('\n')[2], [](start = 28, length = 41)

What about item['example'].split('\n')[2]`? `example is a top level property in our model